tests/test_client/views.py

```diff
@@ -309,5 +310,16 @@ def mass_mail_sending_view(request):
     return HttpResponse("Mail sent")
 
 
+def nesting_exception_view(request):
+    """A simple view that uses a nested client to call another view and
+    then raises an exception
+    """
+    client = Client()
+    data = {'var': '\xf2'}
+    response = client.get('/get_view/', data)
+    raise Exception('exception message')
+    return response
```

Comment:
The bug found is caused by the nested client.get call. If you remove the the stuff above the raise statement, you aren't testing the provided fix.